### PR TITLE
Implement Home UI with list of LAOs (fe2-android)

### PR DESCRIPTION
**Pull request by jbmichel14**

### CODE REVIEW

**Review by celinecamacho — APPROVED** [review]
Great

File: app/src/main/java/com/github/dedis/student20_pop/ui/HomeFragment.java [hunk 1]
```diff
@@ -18,11 +31,106 @@
...
+        }else{
+            welcome.setVisibility(View.GONE);
+            list.setVisibility(View.VISIBLE);
+        }
+        ListView laosListView = view.findViewById(R.id.lao_list);
+        laosListView.setAdapter(new LaoListAdapter(this.getContext()));
+        return view;
+    }
+
+    private class LaoListAdapter extends BaseAdapter {
```
Comment (lhmerino) [inline app/src/main/java/com/github/dedis/student20_pop/ui/HomeFragment.java:64]: What does this class do? Why "Adapter" in the name?

File: app/src/main/java/com/github/dedis/student20_pop/ui/HomeFragment.java [hunk 2]
```diff
@@ -18,11 +31,106 @@
...
+               }else {
+                   startAttendeeUI(lao);
+               }
+            });
+            return convertView;
+        }
+    }
+
+    private ArrayList<Lao> getLaos(){
+        //Now: for testing
```
Comment (lhmerino) [inline app/src/main/java/com/github/dedis/student20_pop/ui/HomeFragment.java:113]: What does this test?

**Review by lhmerino — CHANGES_REQUESTED** [review]
Looks good! Please add some comments to make it clearer to the reader what is the objective of certain code fragments.

File: app/src/main/java/com/github/dedis/student20_pop/ui/HomeFragment.java (hunk 2 above)
Comment (jbmichel14) [reply app/src/main/java/com/github/dedis/student20_pop/ui/HomeFragment.java:113]: It tests the UI with an actual list of LAO's until we implement a way to get real LAO's from the backend

File: app/src/main/java/com/github/dedis/student20_pop/ui/HomeFragment.java [hunk 3]
```diff
@@ -18,11 +31,120 @@
 
     public static final String TAG = HomeFragment.class.getSimpleName();
 
+    //retrieved from the backend, more specifically from the Person's list of Lao's
+    private ArrayList<Lao> laos;
+    //Now: for testing, later: retrieve from Person
```
Comment (AlexisTabin) [inline app/src/main/java/com/github/dedis/student20_pop/ui/HomeFragment.java:36]: Maybe just add a TODO tag so we remember to change that

**Review by AlexisTabin — APPROVED** [review]
Nice work !

File: app/src/main/java/com/github/dedis/student20_pop/ui/HomeFragment.java (hunk 3 above)
Comment (jbmichel14) [reply app/src/main/java/com/github/dedis/student20_pop/ui/HomeFragment.java:36]: done

**Review by lhmerino — APPROVED** [review]

**Pull request merged**